vocode/streaming/synthesizer/cartesia_synthesizer.py

```diff
@@ -52,41 +60,44 @@ def __init__(
             # Cartesia has issues with MuLaw/8000. Use pcm/16000 and
             # create_synthesis_result_from_wav will handle the conversion to mulaw/8000
             self.channel_width = 2
-            self.output_format = "pcm_16000"
-            self.sampling_rate = 16000
+            self.output_format = {
+                "sample_rate": 16000,
+                "encoding": "pcm_s16le",
+                "container": "raw",
+            }
         else:
             raise ValueError(
                 f"Unsupported audio encoding {synthesizer_config.audio_encoding}"
             )
 
+        if not isinstance(self.output_format["sample_rate"], int):
+            raise ValueError(f"Invalid type for sample_rate")
+        self.sampling_rate = self.output_format["sample_rate"]
         self.num_channels = 1
         self.model_id = synthesizer_config.model_id
         self.voice_id = synthesizer_config.voice_id
         self.client = self.cartesia_tts(api_key=self.api_key)
-        self.voice_embedding = self.client.get_voice_embedding(voice_id=self.voice_id)
         
-
     async def create_speech_uncached(
```

Comment:
if interested, would love to support you making this properly streaming!!

would be something like:

```
def chunk_generator(sse):
  async for chunk in sse:
    yield SynthesisResult.ChunkResult(data=chunk)
    
return SynthesisResult(chunk_generator=chunk_generator(generator), lambda seconds: self.get_message_cutoff_from_voice_speed(message, seconds, 150))
```

you would have to also make the synthesizer support mulaw properly (which I think they fixed recently!)